packages/bpk-component-card-list/src/BpkCardListRow/BpkCardListRow-test.tsx

```diff
@@ -0,0 +1,193 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eslint-disable no-underscore-dangle */
+import { useRef } from 'react';
+
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import mockCards from '../../testMocks';
+
+import BpkCardListRow from './BpkCardListRow';
+
+import type { BpkAccessoryTypes } from '../common-types';
+
+jest.mock('../../../bpk-react-utils', () => ({
+  ...jest.requireActual('../../../bpk-react-utils'),
+  isRTL: jest.fn(),
+}));
```

Comment:
What is the requirement of mocking bpk-react-utils here? I can't seem to see usage in this file anywhere